### PR TITLE
RELEASE: 0.2.1-DEMO-XDAI

### PR DESCRIPTION
**Pull request by W3stside**
# DO NOT SQUASH

# [XDAI] (does not work on other networks)
#### Release v0.2.1-DEMO 
xDai release for the demo

### Thanks @fleupold 

Changelog:
* updated contracts to alpha-11
* re-enable xDAI

### CODE REVIEW

File: src/constants/lists.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // the Uniswap Default token list lives here
-export const DEFAULT_TOKEN_LIST_URL = 'tokens.uniswap.eth'
+export const DEFAULT_TOKEN_LIST_URL = 'https://tokens.honeyswap.org'
```
Comment (bh2smith) [inline src/constants/lists.ts:2]: Nice! Do you know if there Is a similar list on rinkeby? Asking for a friend.

**Review by bh2smith — APPROVED** [review]
Looks good! What's up with the do not squash request?

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr86--gpv2swap.review.gnosisdev.com)**: Testing web app

File: package.json [hunk 2]
```diff
@@ -88,6 +87,7 @@
     "start-server-and-test": "^1.11.0",
     "styled-components": "^4.2.0",
     "typescript": "^3.8.3",
+    "uniswap-xdai-sdk": "3.0.3",
```
Comment (Velenir) [inline package.json:90]: `uniswap-xdai-sdk` is both in `devDependencies and dependencies`. Which is it? And which version? `3.0.3`?

**Review by Velenir — APPROVED** [review]
Excellent!

`uniswap-xdai-sdk` is a fork of `@uniswap/sdk`, right? Then maybe we don't need the 100 files with changes in import path only. I'll make a PR

File: src/hooks/useSwapCallback.ts [hunk 3]
```diff
@@ -75,7 +75,7 @@ function useSwapCallArguments(
             feeOnTransfer: false,
             allowedSlippage: new Percent(JSBI.BigInt(allowedSlippage), BIPS_BASE),
             recipient,
-            deadline: deadline.toNumber()
+            ttl: deadline.toNumber()
```
Comment (Velenir) [inline src/hooks/useSwapCallback.ts:78]: Any idea why this change on xDAI?

Comment (W3stside) [reply src/hooks/useSwapCallback.ts:78]: https://github.com/Uniswap/uniswap-sdk/blob/14aeb6eda9992e029866d0e3467bd3d0bba2b25a/src/router.ts

fork likely using an older version

**Comment by W3stside** [comment]
> Excellent!
> 
> `uniswap-xdai-sdk` is a fork of `@uniswap/sdk`, right? Then maybe we don't need the 100 files with changes in import path only. I'll make a PR
@Velenir 
yes if you could change this to not need the external package, fix the `Trade` // other `entities` requiring the `XDAI` appended sdk types that would be amazing. I tried this briefly then realised it's saturday, and went this route for the demo.

Thank you!

File: package.json (hunk 2 above)
Comment (W3stside) [reply package.json:90]: latest

**Comment by Velenir** [comment]
> if you could change this to not need the external package

That is not what I'm talking about. Making xDAI just work with @uniswap/sdk is more involved
I'm talking about dependencies aliases, #88

**Comment by W3stside** [comment]
> Looks good! What's up with the do not squash request?

we squash all feature requests into dev but we merge normally into master

**Comment by Velenir** [comment]
I have a lot of
```
GET https://protocol-xdai.dev.gnosisdev.com/api/v1/tokens/ETH/fee 404 (Not Found)
Error: Error: Error getting the fee
    at updater.ts:45
```

Expected?

**Comment by W3stside** [comment]
> I have a lot of
> 
> ```
> GET https://protocol-xdai.dev.gnosisdev.com/api/v1/tokens/ETH/fee 404 (Not Found)
> Error: Error: Error getting the fee
>     at updater.ts:45
> ```
> 
> Expected?

check the swagger API, returns 404 not found so it's ETH specific as it doesn't exist. not sure the fix rn apart from removing ETH

File: package.json [hunk 4]
```diff
@@ -123,6 +122,7 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "@gnosis.pm/gp-v2-contracts": "^0.0.1-alpha.10"
+    "@gnosis.pm/gp-v2-contracts": "^0.0.1-alpha.11",
+    "@uniswap/sdk": "npm:uniswap-xdai-sdk"
```
Comment (anxolin) [inline package.json:126]: Where can I see where this package comes from. The NPM package points to the original uniswap

is this something meant to be a permanent solution or was a one time for the DEMO thing?

**Review by anxolin — CHANGES_REQUESTED** [review]
Is this PR meant to be merged? it has some hardcoded solution and breaks the principle of not touching src. I understand was just for the DEMO, right?
